.github/workflows/integration.yml

```diff
@@ -253,6 +254,126 @@ jobs:
             sleep 10
           done
           echo LINODE_CLUSTER_CONTEXT=`kubectl config current-context` >> $GITHUB_ENV
+      - name: Set up Wiz
+        if: ${{ inputs.install_profile != 'no-apl' }}
+        run: |
+          # Install required tools for Wiz setup
+          if ! command -v jq &>/dev/null; then
+            sudo apt-get update && sudo apt-get install -y jq
+          fi
+          
+          if ! command -v helm &>/dev/null; then
+            echo "Installing Helm..."
+            HELM_VERSION="v3.14.2"
+            ARCH="amd64"
+            OS="linux"
+            HELM_TAR="helm-${HELM_VERSION}-${OS}-${ARCH}.tar.gz"
+            curl -fsSL -o "$HELM_TAR" "https://get.helm.sh/${HELM_TAR}"
+            tar -zxvf "$HELM_TAR"
+            sudo mv "${OS}-${ARCH}/helm" /usr/local/bin/helm
+            rm -rf "$HELM_TAR" "${OS}-${ARCH}"
+          fi
+
+          # Wiz configuration templates
+          WIZ_KUBERNETES_INTEGRATION_TEMPLATE='
+          global:
+            wizApiToken:
+              clientId: "xudilmphvfal5nrrkhilvzcjjm5cr6unt4wv3gu6uwxsgj6gqcdgm"
+              clientToken: "pZe1En7rjp6bzyWlhkBg1SwiS5efODqdyC3KhWPUEoaN7kNkhqlx6y4xyc1eiZEi"
+              clientEndpoint: "" 
+
+          wiz-kubernetes-connector:
+            enabled: true
+            autoCreateConnector:
+              connectorName: "<connectorName>"
+              clusterExternalId: "<clusterExternalId>"
+            wiz-broker:
+              enabled: true
+
+          wiz-sensor:
+            enabled: true
+            imagePullSecret:
+              create: false
+              name: "sensor-image-pull"
+
+          wiz-admission-controller:
+            enabled: true
+            kubernetesAuditLogsWebhook:
+              enabled: true
+            opaWebhook:
+              enabled: true
+            imageIntegrityWebhook:
+              enabled: false
+              policies:
+                - my-image-trust-policy
+          '
+
+          WIZ_ADMISSION_CONTROL_TEMPLATE='
+          wizApiToken:
+            clientId: "xudilmphvfal5nrrkhilvzcjjm5cr6unt4wv3gu6uwxsgj6gqcdgm"
+            clientToken: "pZe1En7rjp6bzyWlhkBg1SwiS5efODqdyC3KhWPUEoaN7kNkhqlx6y4xyc1eiZEi"
+            clientEndpoint: ""
+          wiz-kubernetes-connector:
+            enabled: true
+          autoCreateConnector:
+            connectorName: "<connectorName>"
+          webhook:
+            clusterExternalId: "<clusterExternalId>"
+          wiz-admission-controller:
+            enabled: true
+          kubernetesAuditLogsWebhook:
+            enabled: true
+          '
```

Comment:
If these are static files, they could also be stored in the repo. `sed` can also be used in these files to make substitutions.